packages/amplify_core/test/datstore/testData/Blog.dart

```diff
@@ -163,7 +166,7 @@ class Blog extends Model {
         key: Blog.POSTS,
         isRequired: false,
         ofModelName: (Post).toString(),
-        associatedKey: Post.BLOG));
+        associatedKey: Post.BLOGID));
```

Comment:
I think these files need to be regenerated with codegen. This field should be `associatedKey: Post.BLOG` with the [testing schema](https://github.com/aws-amplify/amplify-flutter/pull/1599/files#diff-59497bbf708cae3d24c245b38726d4eb4606cd04e2a12d756028651c9447ef9f).